### PR TITLE
Update nginx sample with PHP7 as default

### PR DESCRIPTION
**Pull request by SpacePossum**
welcome PHP7 :)

### CODE REVIEW

File: doc/web_servers.rst [hunk 1]
```diff
@@ -57,7 +57,9 @@ The **minimum configuration** to get your application running under Nginx is:
         # location ~ ^/(index|index_dev)\.php(/|$) {
         location ~ ^/index\.php(/|$) {
             # the ubuntu default
-            fastcgi_pass   unix:/var/run/php5-fpm.sock;
+            fastcgi_pass   unix:/var/run/php/php7.0-fpm.sock;
+            # for running php5
+            # fastcgi_pass   unix:/var/run/php5-fpm.sock;
```
Comment (fabpot) [inline doc/web_servers.rst]: Instead, I would just replace 5 by X. This is just a template afterall.

Comment (SpacePossum) [inline doc/web_servers.rst]: between 5 and 7 the default directory to run the `sock` on changed for Ubuntu from:
`/var/run/` to `var/run/php`
minor diff but still very useful to know imho

Comment (fabpot) [inline doc/web_servers.rst]: But this doc does not assume Ubuntu anywhere. Again, this is just a template, not a ready to go configuration.

Comment (SpacePossum) [inline doc/web_servers.rst]: Far enough, I was thrown off guard with the current `# the ubuntu default` and the `centos` example :+1:

**Comment by fabpot** [comment]
Thank you @SpacePossum.

**Pull request closed without merging**